hw_2/HW2_S_Glenn.r

```diff
@@ -84,10 +87,12 @@ head(vipcls$del_date)
 vipcls$del_date[grep("-1--1--1", vipcls$del_date)] <- NA
 vipcls$del_date <- as.Date(vipcls$del_date)
 ##c. days between enrollment and delivery
-enrollToDelDays <- difftime(vipcls$del_date, vipcls$enroll_date)
+vipcls$enrollToDelDays <- difftime(vipcls$del_date, vipcls$enroll_date)
 ##d. Investigate which ptid has a delivery date prior to enrollment
```

Comment:
Here you weren't originally saving the data to a column in the dataframe, instead you were saving it as a vector. Fixed that here.